docs/architecture/microservices/architect-microservice-container-applications/maintain-microservice-apis.md

```diff
@@ -7,7 +7,7 @@ ms.date: 01/13/2021
 
 A microservice API is a contract between the service and its clients. You'll be able to evolve a microservice independently only if you do not break its API contract, which is why the contract is so important. If you change the contract, it will impact your client applications or your API Gateway.
 
-The nature of the API definition depends on which protocol you're using. For instance, if you're using messaging (like [AMQP](http://www.amqp.org/)), the API consists of the message types. If you're using HTTP and RESTful services, the API consists of the URLs and the request and response JSON formats.
+The nature of the API definition depends on which protocol you're using. For instance, if you're using messaging (like [AMQP](https://www.amqp.org)), the API consists of the message types. If you're using HTTP and RESTful services, the API consists of the URLs and the request and response JSON formats.
```

Comment:
This link doesn't work with https

Reply:
Does it work for you without? I couldn't get it to work either way.

Reply:
No, just tried http and got the same result.